cumulus/node/README.md

```diff
@@ -57,22 +61,20 @@ Now the primary node should be producing blocks.
 Ensure the bootnode for the primary node is correct and run this command to start an executor:
 
 ```bash
-$ ./target/release/subspace-executor \
-    --alice \
-    --collator \
-    --force-authoring \
-    --base-path first-db \
-    --port 40333 \
-    --log=cirrus=trace,txpool=trace,gossip=trace \
-    --rpc-port 8845 \
-    --ws-port 8846 \
+$ ./target/release/subspace-node \
+    --chain dev \
```

Comment:
Why no `--validator` now?

Reply:
It's now unconditionally enabled, so the CLI argument is omitted.
https://github.com/subspace/subspace/blob/fba21ad1355e475a0ddf63fb747464cc04a524be/crates/subspace-node/src/bin/subspace-node.rs#L254

Reply:
Should it though? Full node on execution side doesn't have to be an authority node, can just validate things that are happening.

Reply:
It's a known issue that we require the embedded primary node to an authority to get the new slots notification.

https://github.com/subspace/subspace/blob/fba21ad1355e475a0ddf63fb747464cc04a524be/crates/subspace-node/src/bin/subspace-node.rs#L252-L254